### DISCUSSION TITLE
basic question

### DISCUSSION THREAD

**Discussion by lxdmay** [svenreiche/Genesis-1.3-Version4#103] [Q&A]
1. the unit of delgam is % ? delgam = σr/r0 (σr is standard deviation and r0 is gamma0 in the Global group ).
2. how to set slen correctly? when I set slen small, the last few lines of some two-dimensional arrays (such power) become zero? So is there any matching relationship between setting slen correctly?
3. When I set I(s), the current of the last few slices is becoming zero. Is it because I set too many slice values?
Please answer these basic questions.

**Comment by svenreiche** [comment]
1.  The unit is the rms spread in terms of the electron rest mass. So a value of delgam=1 corresponds to an energy spread of 511keV
2.  + 3.  In the parallel execution of Genesis with mpirun, the core requires a symmetric number of slices per node. So if theslen in the time namelist requires 7 slices but one starts mpirun with 4 cores than genesis is filling up to 8 slices. That the reason tjhat the slice number is not always consistent with the time namelist.

**Reply by lxdmay** [reply]
Thank you
